types/react-navigation/index.d.ts

```diff
@@ -651,3 +652,22 @@ export interface NavigationScreenProps<Params> {
 /**
  * END CUSTOM CONVENIENCE INTERFACES
  */
+
+
+/*
```

Comment:
Can you move all of this up into the `CUSTOM CONVENIENCE INTERFACES` area (between the `BEGIN` and `END` marks)? Maybe put this next to the stack navigator stuff in that area?